lib/pbench/server/database/models/datasets.py

```diff
@@ -833,12 +848,16 @@ def validate(dataset: Optional[Dataset], key: str, value: Any) -> Any:
         should be used in most contexts where client-visible metadata paths
         are used.
 
-        1) For 'dataset.name', we require a non-empty UTF-8 encoded string.
-        2) For 'server.deletion', the string must be an ISO date/time string,
-           and we fail otherwise. We store only the UTC date, as we don't
-           guarantee that deletion will occur at any specific time of day. The
-           maximum retention period (from upload) is defined by the server
-           configuration dataset-lifetime property.
+        1) 'dataset.name': a non-empty UTF-8 encoded string.
+        2) 'server.deletion': an ISO date/time string. We store only the UTC
+           date, as we don't guarantee that deletion will occur at any specific
+           time of day. The maximum retention period (from upload) is defined
+           by the server configuration dataset-lifetime property.
+        3) 'server.archiveonly': a boolean value. When true, the server will
+           not unpack or index the dataset. (Only meaningful on upload.)
+        4) 'server.origin': a string that the put-shim 0.69 server will use
+           to record the name of the 0.69 satellite server from which the
+           dataset is relayed.
```

Comment:
Less is more? Or is more actually less?

Reply:
I think more is actually less; therefore, less is better, if not more.

(This is an instance of _Jack's Law_ -- if you record the same information in more than one place, then (eventually) at least one of them will be wrong -- it would be better if this referred to the previous description, either explicitly or by simply omitting redundancy here.)